app/gateway/2.7.x/install-and-run/docker.md

```diff
@@ -153,61 +187,79 @@ docker pull kong:{{page.kong_versions[page.version-index].ce-version}}-alpine
+ ```
 {% endnavtab %}
 {% endnavtabs %}
 {% endcapture %}
 {{ start_container | indent | replace: " </code>", "</code>" }}
 
-   {:.note}
-   > **Note**: The `HOSTNAME` for `KONG_PORTAL_GUI_HOST` should be preceded by a protocol, for example, `http://`.
-
-2. Verify your installation:
+    Where:
+    * `--name` and `--network`: The tag of the {{site.base_gateway}} image that
+    you're using, and the Docker network it communicates on.
+    * [`KONG_DATABASE`](/gateway/{{page.kong_version}}/reference/configuration/#database):
+    Specifies whether this node connects directly to a database.
+    * [`KONG_PG_HOST`](/gateway/{{page.kong_version}}/reference/configuration/#postgres-settings):
+    The name of the Postgres Docker container that is communicating over the
+    `kong-net` network.
+    * [`KONG_PG_USER` and `KONG_PG_PASSWORD`](/gateway/{{page.kong_version}}/reference/configuration/#postgres-settings):
+     The Postgres username and password. Kong Gateway needs the login information
+     to store configurations in this database.
+    * All [`_LOG`](/gateway/{{page.kong_version}}/reference/configuration/#general-section)
+    parameters: set filepaths for the logs to output to, or use the values in
+    the example to  print messages and errors to `stdout` and `stderr`.
+    * [`KONG_ADMIN_LISTEN`](/gateway/{{page.kong_version}}/reference/configuration/#admin_listen):
+    The port that the Kong Admin API listens on for requests.
+    * [`KONG_ADMIN_GUI_URL`](/gateway/{{page.kong_version}}/reference/configuration/#admin_gui_url):
+    (Enterprise only) The URL for accessing Kong Manager, preceded by a protocol
+    (for example, `http://`).
+
+1. Verify your installation:
 
     Access the `/services` endpoint using the Admin API:
 
-    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/services</code></pre>
+    ```sh
+    curl -i -X GET --url http://localhost:8001/services
+    ```
 
     You should receive a `200` status code.
 
-3. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
-
-      <pre><code>http://<div contenteditable="true">{HOSTNAME}</div>:8002</code></pre>
+1. (Not available in OSS) Verify that Kong Manager is running by accessing it using the URL specified in `KONG_ADMIN_GUI_URL`:
 
-## Install Gateway in DB-less mode
+    ```
+    http://localhost:8002
+    ```
 
-{:.important}
-> **Important**: Running {{site.base_gateway}} on Docker in DB-less mode is
-supported for both Enterprise and OSS images. However, there is currently no
-documentation for setting up an Enterprise image in DB-less mode, so adjust the
-following steps as needed for an Enterprise deploy.
+## Install Kong Gateway in DB-less mode
 
 The steps involved in starting Kong in [DB-less mode](/gateway/{{page.kong_version}}/reference/db-less-and-declarative-config) are the following:
 
-1. Create a Docker network:
+### Set up a Docker volume
```

Comment:
This title talks about `volume` then the first step is to create a network, which is confusing to me